cmd/metrics-node-sampler/integration/testdata/basic/expected.json

```diff
@@ -57,62 +57,14 @@
 			"avgCPUThrottledNanoSec": "70",
 			"avgCPUPeriodsSec": "800",
 			"avgCPUThrottledPeriodsSec": "130",
-			"networkRxBytes": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkRxPackets": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkRxErrors": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkRxDropped": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkTxBytes": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkTxPackets": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkTxErrors": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
-			"networkTxDropped": [
-				"0",
-				"0",
-				"0",
-				"0",
-				"0"
-			],
+			"networkRxBytes": [],
```

Comment:
Now that none of the tests have values in the results, are we sure the code to add the data will actually work on `nil` slices?

Reply:
Do you think this is not suffiient: https://github.com/kubernetes-sigs/usage-metrics-collector/pull/113/files#diff-8ecbe56b8ca855c1d104dbc7887279bc1bdf7160ba43974448734237e846caad